ddtrace/contrib/unittest/patch.py

```diff
@@ -258,7 +258,22 @@ def _extract_test_file_name(item) -> str:
 
 def _extract_module_file_path(item) -> str:
     if _is_test(item):
-        return os.path.relpath(inspect.getfile(item.__class__))
+        try:
+            test_module_object = inspect.getfile(item.__class__)
+        except TypeError:
+            log.debug(
+                "Tried to collect module file path but it is a built-in Python function",
+            )
+            return ""
+        try:
+            module_file_path = os.path.relpath(test_module_object, start=os.getcwd())
+            return module_file_path
+        except ValueError:
+            log.debug(
+                "Tried to collect module file path but it is using different drive paths on Windows, "
+                "using absolute path instead"
+            )
+            return os.path.abspath(test_module_object)
```

Comment:
I think we're reusing this pattern (`try: replath, except ValueError`) enough times that we should probably roll it into its own utility function.

It'd be nice to get this fix out sooner rather than later, though, so I'm open to making a follow-up PR to clean up.